### DISCUSSION TITLE
Matrix Field: Tooltip in Editor

### DISCUSSION THREAD

**Discussion by craftedsystems** [craftcms/cms#11784] [Ideas] [answered]
A tooltip in the matrix editor would be great. Otherwise long names can not be identified without clicking the cogwheel.

<img width="220" alt="Screenshot 2022-08-16 at 10 59 39" src="https://user-images.githubusercontent.com/26137287/184840815-a46e030f-c4f1-4d48-b157-919775f1887a.png">

**Comment by brandonkelly** [comment, marked as answer]
Good idea. Just added that for the next Craft 3 and 4 releases, for both block types and their sub-fields. (5a7c2c69eb8db725c759b23f94674b909c9e3184)
